index.html

```diff
@@ -157,21 +157,24 @@ <h2>Contact me</h2>
                 I'd love to help with it </p>
             </div>
             <div class="contact-form">
-            <form action="https://formspree.io/f/xnqwordb" method="post">
+            <form id="form" class="form" action="https://formspree.io/f/xnqwordb" method="post">
 
-                <input name="first_name" 
+                <input class="form-input" name="first_name" 
                 id="text" type="text" 
                 maxlength="30" 
                 placeholder="Afolabi" required>
               
-                <input name="Email" 
+                <input class="form-input" name="Email" 
                 id="email" type="email" 
                 placeholder="example@gmail.com" required>
-
-                <textarea id="message" 
+                
+                <textarea class="form-input" id="message" 
                 name="message" maxlength="500" required>Write your message here
                 </textarea>
 
+                <div class="error-text hide">
+                    <em>Failed, kindly enter your email in lower case</em>
+                 </div>
                 <button type="submit" class="btn">Get It Touch</button>
```

Comment:
- [ ] Great job in implementing the changes, the error message appears when clicking on submit button, however, it also appears when clicked on text area. As suggested previously, kindly make sure your error message is only triggered when the user clicks on the submit button.

https://user-images.githubusercontent.com/70269507/169642954-3e56148e-f4c7-4476-9e3c-d38ee8dafc3b.mov

Reply:
okay thanks 
have done has you requested